### PR TITLE
Milestone 4

### PR DESCRIPTION
**Pull request by sergiocortessat**
Pull request 3

Hello TSE. In this milestone, I implemented the logic.js, calculate, and some code to the app.js. As such, it can now perform operations and display the results on the screen. 

- [x]  operate
- [x] calculate
- [x] Added calculate to app.js

### CODE REVIEW

File: src/Logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    if (operation === '+') {
+      result = new Big(numberOne).plus(new Big(numberTwo));
+    } else if (operation === '-') {
+      result = new Big(numberOne).minus(new Big(numberTwo));
+    } else if (operation === 'X') {
+      result = new Big(numberOne).times(new Big(numberTwo));
+    } else if (operation === '/') {
+      try {
+        result = new Big(numberOne).div(new Big(numberTwo));
+      } catch (error) {
```
Comment (bellom) [inline src/Logic/operate.js:17]: # [Optional]
Your code appears to be WET-(Write Every Time), Microverse encourage DRY-(Don't Repeat Yourself) code.
In your case, you repeated `new Big`.
You could have it in a variable then reuse it.
example:
```
    const firstNum = Big(numberOne);
    const secondNum = Big(numberTwo);
```

File: src/Components/App.js [hunk 2]
```diff
@@ -1,14 +1,55 @@
...
+    } else if (operation === '+/-') {
+      if (next === null || next === '0') {
+        result = total;
+      } else {
+        result = next;
+      }
+    } else if (operation === '%') {
+      result = operation ? total : `${total}%`;
+    } else if (operation !== '%') {
+      result = next ? total + operation + next : total + operation;
```
Comment (bellom) [inline src/Components/App.js:43]: # [Optional]
### Heads-up:

There is `logic/calculate.js` file that you're supposed to have this logic, moreover, I think you already have it there, so you don't need to put it in here, and also these pieces of code are supposed to be implemented in the next milestone.

**Review by bellom — APPROVED** [review]
## Hi  @sergiocortessat :clap:

##### You have done wonderfully well, you passed all checklist for this milestone :+1:
 
# Status: Approved 👍 

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit the project completion form.
Good luck with your next milestone :slightly_smiling_face:_ 


**Best**

File: src/Logic/operate.js (hunk 1 above)
Comment (sergiocortessat) [reply src/Logic/operate.js:17]: Done

File: src/Components/App.js (hunk 2 above)
Comment (sergiocortessat) [reply src/Components/App.js:43]: Done. Understood

**Pull request merged**